### PR TITLE
Updating homepage & adding highlights

### PR DESCRIPTION
**Pull request by rhoslynroebuck**
Added an overview of NanoVer to the homepage and moved the sections that were there into their own "Highlights" page (with some images + videos).

The aim is to make the homepage brief and clean and pointing the reader to the relevant parts of the docs. 

Harry and I also discussed getting rid of the list of contents on the homepage. What do we think? Cause it's super cluttered and the contents is on the sidebar anyway. If we don't have too many headings on the sidebar and they are appropriately named, then I think a full list of contents isn't needed (or at least not on the homepage).

This is just a suggestion, totally up for discussion!

### CODE REVIEW

**Comment by Ragzouken** [comment]
Keeping things clean and simple sounds good to me. I agree we don't need an exhaustive contents that mirrors the sidebar but I think it's good to have some prompting for good places to continue reading--your sections end with a link to read further, which covers that. So it looks good to me.

(btw is there an easy way to preview it as a web page, or do I need to checkout and build myself?)

**Comment by rhoslynroebuck** [comment]
That's a good question! I'm not sure if there's a nicer way, but here's the build directory for you

[build.zip](https://github.com/user-attachments/files/16245381/build.zip)

File: source/index.rst [hunk 1]
```diff
@@ -1,53 +1,35 @@
 Welcome to NanoVer's documentation!
 ===================================
 
-* `Download the VR client <https://github.com/IRL2/nanover-imd/releases/download/nightly/StandaloneWindows64.zip>`_
-* :ref:`Install the software <Installation>`
-* :ref:`Read tutorials <Tutorials>`
+NanoVer is a free, open-source and flexible software that can be used as:
```
Comment (Mohamed-Dhouioui) [inline source/index.rst:4]: Do we want to state that it is under the **MIT license** and reference it?

**Review by Mohamed-Dhouioui — COMMENTED** [review]
I've added some comments in where I think improvements could be done.

File: source/index.rst (hunk 1 above)
Comment (rhoslynroebuck) [reply source/index.rst:4]: This was moved to the Highlights section, but I'll put it back here as well.

File: source/index.rst [hunk 2]
```diff
@@ -1,53 +1,35 @@
...
-* `Get the source code of the VR client <https://github.com/IRL2/nanover-imd>`_
+As a client-server application, NanoVer consists of:
 
+(1) **The NanoVer server**: the server communicates with a physics engine, e.g. OpenMM, to facilitate interactive MD simulations.
+(2) **Nanover IMD**: a client that can connect to a NanoVer server to access simulation data and facilitate user interaction with the molecular system. This can be done using:
 
-Use insightful visuals
-----------------------
+    * A **python client**: a python script, e.g. a Jupyter notebook. Check out the :ref:`tutorials <Tutorials>` for further information.
+    * A **VR client**: an instance of the NanoverIMD program (available `here <https://github.com/IRL2/nanover-imd/releases/download/nightly/StandaloneWindows64.zip>`_ as an executable) that enables a user to visualise and interact with the real-time MD simulation in VR.
```
[the remark this answers is not in the document]
Comment (rhoslynroebuck) [reply source/index.rst:17]: Good idea, I've changed this in the next commit

**Review by Mohamed-Dhouioui — APPROVED** [review]

**Review by hjstroud — APPROVED** [review]
Looks great to me, we discussed and finalised some details in person and I'm happy with the changes that have been made since. Good job!

**Pull request merged**